src/widgets/_menus.scss

```diff
@@ -5,6 +5,16 @@
 
     separator {
         min-height: rem(6px);
+
+        &.horizontal {
+            border-top: 1px solid rgba(black, 0.15);
```

Comment:
I think this probably needs to be added to index as `menu_separator` so that we can make sure to keep it consistent

Reply:
Adding it there.